frontend/src/app/modules/boards/boards-root/boards-root.component.ts

```diff
@@ -1,12 +1,30 @@
-import {Component} from "@angular/core";
-import {BoardCacheService} from "core-app/modules/boards/board/board-cache.service";
+import {Component, Injector} from "@angular/core";
+import {BoardConfigurationService} from "core-app/modules/boards/board/configuration-modal/board-configuration.service";
+import {BoardActionsRegistryService} from "core-app/modules/boards/board/board-actions/board-actions-registry.service";
+import {BoardStatusActionService} from "core-app/modules/boards/board/board-actions/status/status-action.service";
+import {BoardVersionActionService} from "core-app/modules/boards/board/board-actions/version/version-action.service";
+import {QueryUpdatedService} from "core-app/modules/boards/board/query-updated/query-updated.service";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/79acb5e1b7595d25c11c1733b329ae492e5b644e/frontend/src/app/modules/boards/boards-root/boards-root.component.ts#L7"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>